file1.js

```diff
@@ -0,0 +1,8 @@
+/* global QamineUtils */
+
+"use strict";
+
+(function($, window) {
+  $('.jsSelectedMainEmail').text(postVariable.email);
```

Comment:
![Codacy](https://app.staging.codacy.org/assets/images/favicon.png) Issue found: ['postVariable' is not defined. (no-undef)](https://app.staging.codacy.org/app/codacytestuser/CheckPrCommentsPublicProjectStaging/pullRequest?prid=4277)